go.mod

```diff
@@ -1,3 +1,14 @@
 module billion-rows
 
-go 1.24
+go 1.24.6
```

Comment:
should really be 1.24.0  or 1.24 here

Reply:
I guess it can just be 1.24 and it just uses the latest?

Reply:
sorta, this is the language requirement not what is actually used to build (which is specified in the action)